code/__DEFINES/tools.dm

```diff
@@ -1,6 +1,7 @@
-#define TOOL_NONE 0
-#define TOOL_CROWBAR 1
-#define TOOL_MULTITOOL 2
-#define TOOL_SCREWDRIVER 3
-#define TOOL_WIRECUTTER 4
-#define TOOL_WRENCH 5
\ No newline at end of file
+// Tool types
+#define TOOL_CROWBAR 		"crowbar"
+#define TOOL_MULTITOOL 		"multitool"
+#define TOOL_SCREWDRIVER 	"screwdriver"
+#define TOOL_WIRECUTTER 	"wirecutter"
+#define TOOL_WRENCH 		"wrench"
+#define TOOL_WELDER 		"welder"

```

Comment:
Why change these defines? Removing `TOOL_NONE` on its own is probably well enough, but at a glance you don't seem to be using these specific strings anywhere. If they're magic values they might as well stay numbers

Reply:
Commit message says it well enough: strings can be list indexes, numbers can't. This isn't used in this PR, but I have two more in the pipeline, and at least one depends on it.